### ISSUE TITLE
Converting a component with `convert` not returning a correct type

### ISSUE THREAD

**Issue by chanlito** [wonderful-panda/vue-tsx-support#13]
In 2.1.1, using the below code:

```ts
export const VAvatar = ofType<Props, Events, ScopedSlots>().convert(
  VueAvatarOriginal,
);
```

the type of `VAvatar` is like the following:

```ts
const VAvatar: VueConstructor<{
    _tsxattrs: TsxComponentAttrs<Props, Events, ScopedSlots>;
} & Vue>
```

but when I use with 2.2.0

the type of `VAvatar` is like the following instead:

```ts
const VAvatar: VueConstructor<any>
```

**Comment by chanlito** [comment]
I've made a repo here https://github.com/chanlito/vue-tsx-support-convert-type-bug showing this issue.

**Comment by wonderful-panda** [comment]
Sorry, I have no time to fix this problem now.

This is a workaround for now.

```typescript
const Avatar = ofType<Props, Events>().convert(require('vue-avatar').Avatar as typeof Vue);
```

**Comment by wonderful-panda** [comment]
Fixed in v2.2.1

**Comment by pksorensen** [comment]
@wonderful-panda - is this related https://github.com/wonderful-panda/vue-tsx-support/issues/48
